<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GeometrySystemControllerForm</class>
 <widget class="QWidget" name="GeometrySystemControllerForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>338</width>
    <height>822</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>6</number>
   </property>
   <property name="topMargin">
    <number>6</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>4</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QWidget" name="widget_settingsButtonsPlaceholder" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <layout class="QGridLayout" name="_19">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <property name="spacing">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <widget class="QPushButton" name="pushButton_settings_display">
          <property name="toolTip">
           <string>Configure overall display options such as phi-sector cutaways and transparency</string>
          </property>
          <property name="text">
           <string>Display</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="pushButton_settings_interactions">
          <property name="toolTip">
           <string>Configure what happens when a geometry volume is clicked</string>
          </property>
          <property name="text">
           <string>Interactions</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QPushButton" name="pushButton_settings_iconisedvols">
          <property name="toolTip">
           <string>Show a list of iconised/zapped volumes and possible restore them.</string>
          </property>
          <property name="text">
           <string>Icon Box</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="pushButton_settings_misc">
          <property name="toolTip">
           <string>Miscellaneous controls such as automatic actions on multiple volumes, or editing the colour of the last volume clicked.</string>
          </property>
          <property name="text">
           <string>Misc.</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="pushButton_settings_treebrowser">
          <property name="toolTip">
           <string>Browse the detector volume in a tree-view.</string>
          </property>
          <property name="text">
           <string>Browser</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QPushButton" name="pushButton_settings_unused">
          <property name="toolTip">
           <string>Currently unused</string>
          </property>
          <property name="text">
           <string>Unused</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>8</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="_2">
       <property name="spacing">
        <number>4</number>
       </property>
       <item>
        <widget class="QGroupBox" name="groupBox_neutrino">
         <property name="title">
          <string>FaserNu</string>
         </property>
         <layout class="QVBoxLayout" name="_12">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>4</number>
          </property>
          <property name="topMargin">
           <number>4</number>
          </property>
          <property name="rightMargin">
           <number>4</number>
          </property>
          <property name="bottomMargin">
           <number>4</number>
          </property>
          <item>
           <layout class="QHBoxLayout" name="_14">
            <item>
             <widget class="QCheckBox" name="checkBox_Emulsion">
              <property name="text">
               <string>Emulsion</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>1</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>       
        <widget class="QGroupBox" name="groupBox_scintillator">
         <property name="title">
          <string>Scintillator</string>
         </property>
         <layout class="QHBoxLayout" name="_3">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>4</number>
          </property>
          <property name="topMargin">
           <number>4</number>
          </property>
          <property name="rightMargin">
           <number>4</number>
          </property>
          <property name="bottomMargin">
           <number>4</number>
          </property>
          <item>
           <layout class="QGridLayout" name="_4">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <property name="horizontalSpacing">
             <number>6</number>
            </property>
            <property name="verticalSpacing">
             <number>0</number>
            </property>
            <item row="0" column="0">
             <widget class="QCheckBox" name="checkBox_VetoNu">
              <property name="text">
               <string>VetoNu</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QCheckBox" name="checkBox_Veto">
              <property name="text">
               <string>Veto</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QCheckBox" name="checkBox_Trigger">
              <property name="text">
               <string>Trigger</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QCheckBox" name="checkBox_Preshower">
              <property name="text">
               <string>Preshower</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <spacer>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>1</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_tracker">
         <property name="title">
          <string>Tracker</string>
         </property>
         <layout class="QHBoxLayout" name="_5">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>4</number>
          </property>
          <property name="topMargin">
           <number>4</number>
          </property>
          <property name="rightMargin">
           <number>4</number>
          </property>
          <property name="bottomMargin">
           <number>4</number>
          </property>
          <item row="0" column="0">
           <widget class="QCheckBox" name="checkBox_SCT">
            <property name="text">
             <string>SCT</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QCheckBox" name="checkBox_dipole">
            <property name="text">
             <string>Dipole magnets</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>1</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_calorimeter">
         <property name="title">
          <string>Calorimeter</string>
         </property>
         <layout class="QVBoxLayout" name="_121">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>4</number>
          </property>
          <property name="topMargin">
           <number>4</number>
          </property>
          <property name="rightMargin">
           <number>4</number>
          </property>
          <property name="bottomMargin">
           <number>4</number>
          </property>
          <item>
           <layout class="QHBoxLayout" name="_141">
            <item>
             <widget class="QCheckBox" name="checkBox_Ecal">
              <property name="text">
               <string>Ecal</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>1</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_misc">
         <property name="title">
          <string>Miscellaneous</string>
         </property>
         <layout class="QVBoxLayout" name="_12_2">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>4</number>
          </property>
          <property name="topMargin">
           <number>4</number>
          </property>
          <property name="rightMargin">
           <number>4</number>
          </property>
          <property name="bottomMargin">
           <number>4</number>
          </property>
          <item>
           <layout class="QHBoxLayout" name="_13">
            <item>
             <widget class="QCheckBox" name="checkBox_CavernInfra">
              <property name="text">
               <string>Cavern Infrastructure</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QCheckBox" name="checkBox_other">
            <property name="text">
             <string>Other unrecognised volumes.</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="_15">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <layout class="QVBoxLayout" name="_16">
           <property name="spacing">
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="label_12">
             <property name="text">
              <string>Instructions:</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="_17">
             <property name="spacing">
              <number>0</number>
             </property>
             <item>
              <spacer>
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeType">
                <enum>QSizePolicy::Fixed</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <layout class="QVBoxLayout" name="_18">
               <property name="spacing">
                <number>1</number>
               </property>
               <item>
                <widget class="QLabel" name="label_9">
                 <property name="text">
                  <string>Crtl+click: Expand to child volumes</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_10">
                 <property name="text">
                  <string>Shift+click: Contract to mother volume</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_11">
                 <property name="text">
                  <string>Z+click: Iconify (remove) volume</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>1</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <spacer>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>1</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>1</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
